account_payment_mode/migrations/9.0.1.0.0/post-migration.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Sergio Teruel <sergio.teruel@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+def recompute_acc_type(env):
+    """Force recompute acc_type, issue with inheritance chain between base
+    module and base_iban with this module"""
+    if openupgrade.is_module_installed(env.cr, 'base_iban'):
+        partner_banks = env['res.partner.bank'].search([])
+        for partner_bank in partner_banks:
+            partner_bank.write({'acc_number': partner_bank.acc_number})
```

Comment:
Can you call _compute_acc_type directly?
`partner_banks = env['res.partner.bank'].search([])._compute_acc_type()`

or at least remove for:
`partner_banks.write({'acc_number': partner_bank.acc_number})`

Reply:
Sorry, forget remove for